streampark-e2e/streampark-e2e-case/src/test/java/org/apache/streampark/e2e/cases/RoleManagementTest.java

```diff
@@ -0,0 +1,122 @@
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@StreamPark(composeFiles = "docker/basic/docker-compose.yaml")
+public class RoleManagementTest {
+  private static RemoteWebDriver browser;
+
+  private static final String userName = "admin";
+
+  private static final String password = "streampark";
+
+  private static final String teamName = "default";
+
+  private static final String newRoleName = "new_role";
+
+  private static final String newDescription = "new_description";
+
+  private static final String existMenuName = "Apache Flink";
+
+  @BeforeAll
+  public static void setup() {
+    new LoginPage(browser)
+        .login(userName, password, teamName)
+        .goToNav(SystemPage.class)
+        .goToTab(RoleManagementPage.class);
+  }
+
+  @Test
+  @Order(10)
+  void testCreateUser() {
+    final RoleManagementPage roleManagementPage = new RoleManagementPage(browser);
+    roleManagementPage.createRole(newRoleName, newDescription, existMenuName);
+
+    Awaitility.await()
+        .untilAsserted(
+            () ->
+                assertThat(roleManagementPage.roleList())
+                    .as("Role list should contain newly-created role")
+                    .extracting(WebElement::getText)
+                    .anyMatch(it -> it.contains(newRoleName)));
+  }
+
+  @Test
+  @Order(20)
+  void testCreateDuplicateRole() {
+    final RoleManagementPage roleManagementPage = new RoleManagementPage(browser);
+    roleManagementPage.createRole(newRoleName, newDescription, existMenuName);
+
+    Awaitility.await()
+        .untilAsserted(
+            () ->
+                assertThat(roleManagementPage.errorMessageList())
+                    .as("Role Name Duplicated Error message should be displayed")
+                    .extracting(WebElement::getText)
+                    .anyMatch(it -> it.contains("Sorry, the role name already exists")));
+
+    roleManagementPage.createRoleForm().buttonCancel().click();
+  }
+    @Test
+    @Order(30)
+    void testEditRole() {
+      final RoleManagementPage roleManagementPage = new RoleManagementPage(browser);
+
+      String newEditDescription = newDescription + "_edit";
+      String newEditMenuName = "System";
+      roleManagementPage.editRole(newRoleName, newEditDescription, newEditMenuName);
+
+    //TODO: there is no description filed value actual exist
+    //       Awaitility.await()
+    //           .untilAsserted(
+    //               () ->
+    //                   assertThat(roleManagementPage.roleList())
+    //                       .as("Role list should contain newly-created role")
+    //                       .extracting(WebElement::getText)
+    //                       .anyMatch(it -> it.contains(newEditDescription)));
+    }
```

Comment:
I tested it and it was a bug. The description did not showing successfully. Would you like to fix it?

Reply:
> I tested it and it was a bug. The description did not showing successfully. Would you like to fix it?

i will fix it.

Reply:
You can open another to fix this bug and uncomment this test in that PR.